fluss-server/src/main/java/org/apache/fluss/server/metrics/group/TableMetricGroup.java

```diff
@@ -43,6 +44,10 @@ public class TableMetricGroup extends AbstractMetricGroup {
 
     private final Map<TableBucket, BucketMetricGroup> buckets = new HashMap<>();
 
+    // Directly manage RocksDB metrics for aggregation
+    // This is cleaner than passing through BucketMetricGroup
+    private final Map<TableBucket, RocksDBMetrics> rocksDBMetricsMap = new HashMap<>();
```

Comment:
Both this `rocksDBMetricsMap` and `buckets` maps should be `ConcurrentHashMap`, because they may be invoked/iterated in other threads (metric reporter).